### ISSUE TITLE
Filter duplicate comments

### ISSUE THREAD

**Issue by lukebp** [decred/politeiagui#1533]
decred/politeia#969 prevents duplicate comments from being submitted, like what happened in this thread:

https://proposals.decred.org/proposal/65bde4146b845e7e839d6916d4d8f642bc39c250df5379c2f1e26c4ab778ec1a/comments/5

This does not however do anything to remove/filter the existing duplicate comments.  This looks bad and @marcopeereboom wants it fixed.

Duplicate comments will have the same signature.  You can filter out the duplicate comments in the gui by only displaying one comment with that specific signature.

**Comment by lukebp** [comment]
Looks like this is the root cause of the issue.  We need to verify that this doesn't exist in the redesign.

https://proposals.decred.org/proposal/65bde4146b845e7e839d6916d4d8f642bc39c250df5379c2f1e26c4ab778ec1a/comments/15

**Comment by fernandoabolafio** [comment]
I was looking at the comments response for the ammarooni proposal and I noticed that from the 7 duplicated comments, 6 have the same signature while 1 doesn't. Perhaps filtering by the comment text and the signature would be a more reliable approach.

> Looks like this is the root cause of the issue. We need to verify that this doesn't exist in the redesign.

That has been addressed. While the comment is being posted, the button is blocked and the form is cleared immediately after the request success.
